src/fhirtypes/ElementDefinition.ts

```diff
@@ -1083,11 +1083,20 @@ export class ElementDefinition {
         // References always have a code 'Reference' w/ the referenced type's defining URL set as
         // one of the targetProfiles.  If the targetProfile property is null, that means any
         // reference is allowed.
+        // When 'Reference' keyword is used, prefer to match on the 'Reference' type over the
+        // 'CodeableConcept' type if they both exist on the element.
```

Comment:
Think you mean `'CodeableReference`' in this comment 🙂.

Reply:
Updated in [11a10a4](https://github.com/FHIR/sushi/pull/1056/commits/11a10a4d828fe0b790e6dff158a36c538f505df6)!